telemetry/packages/server/src/modules/influx/Influx.service.ts

```diff
@@ -1,43 +1,69 @@
 import { InfluxDB, QueryApi, WriteApi } from '@influxdata/influxdb-client';
-import { Injectable, OnModuleInit } from '@nestjs/common';
+import { Injectable, LoggerService, OnModuleInit } from '@nestjs/common';
 import {
   INFLUX_TELEMETRY_BUCKET,
   INFLUX_ORG,
   INFLUX_TOKEN,
   INFLUX_URL,
   INFLUX_FAULTS_BUCKET,
 } from '../core/config';
+import { Logger } from '../logger/Logger.decorator';
```

Comment:
Not knowing anything about TS, is it normal to use relative paths?

Reply:
Yeah it's pretty normal, but could convert them all to absolute if you want?

Reply:
if this is how you do them idm. I prefer absolute generally, cos it makes it easier to work out where things are but I'll bow to whatever the existing standard is